### PR TITLE
[New] Add grapesjs w/ npm auto-update

### PR DESCRIPTION
**Pull request by extend1994**
Pull request for issue: #11700

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/artf/grapesjs
 * Official website (optional, not the repository): https://grapesjs.com
 * NPM package url (optional): https://www.npmjs.com/package/grapesjs
 * License and its reference: [BSD-3-Clause](https://github.com/artf/grapesjs/blob/dev/LICENSE)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 144
   - Count of stars: 2,228
   - Count of forks: 336
 * NPM download stats (optional):
   - Downloads in the last day: 603
   - Downloads in the last week: 1,437
   - Downloads in the last month: 3,247

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by extend1994** [comment]
1. Use the rule `**/!(fontawesome*|FontAwesome*)` to exclude the existed library FontAwesome under `fonts`
2. Use "/" for not continuous versions and use "~" for continuous versions in the second commit message to describe the added versions.

File: ajax/libs/grapesjs/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    "wysiwyg",
+    "template",
+    "editor"
+  ],
+  "npmName": "grapesjs",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/!(fontawesome*|FontAwesome*)"
```
Comment (sufuf3) [inline ajax/libs/grapesjs/package.json]: How about this `**/!(fontawesome|FontAwesome)*`?
What do you think?
Thank you.

**Comment by extend1994** [comment]
Updated, please review again, thanks.

**Comment by artf** [comment]
thanks @extend1994

**Comment by extend1994** [comment]
No problem @artf 😄

**Comment by extend1994** [comment]
ping @maruilian11 @pvnr0082t

**Comment by sufuf3** [comment]
ping @maruilian11 @pvnr0082t

**Comment by extend1994** [comment]
@maruilian11 Thanks for suggestion. I remove the duplicated words. Please help me review it, thank you.

**Pull request merged**